src/sentry/static/sentry/app/views/eventsV2/eventQueryParams.tsx

```diff
@@ -141,7 +141,7 @@ export const FIELDS: {[key: string]: ColumnValueType} = {
   'transaction.duration': 'duration',
   'transaction.op': 'string',
   apdex: 'number',
-
+  impact: 'number',
```

Comment:
If impact needs to be plotted on the graph it needs to be added to the AGGREGATE_FIELD_ALIAS list as well.